ghost/core/core/server/web/parent/app.js

```diff
@@ -23,6 +23,13 @@ module.exports = function setupParentApp() {
     // @TODO: figure out if this is really needed everywhere? Is it not frontend only...
     parentApp.use(mw.ghostLocals);
 
+    // Enable request queuing if configured
+    const queueConfig = config.get('optimization')?.requestQueue;
```

Comment:
Is your thought to have this as our 'on' switch for testing vs. having a different build? Makes sense to me.

Reply:
Yep, means we don't have to worry about builds getting swapped out on new releases 👍